scripts/dnd5e-helpers.js

```diff
@@ -2637,7 +2637,7 @@ class CoverData {
       }
       /** whisper the message if we are being a cautious GM */
       let recipients;
-      if (game.user.isGM && game.settings.get('dnd5e-helpers', 'losMaskNPCs')) {
```

Comment:
I think this could actually be removed entirely, and just wisper to the GM by default